style.css

```diff
@@ -0,0 +1,129 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+  letter-spacing: 1.2;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  padding-bottom: 4rem;
+  background-color: #fff;
+  border-bottom-left-radius: 12rem;
+  background-image: url(./images/mobile-background-img.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top 8rem right 0;
+}
```

Comment:
- [Optional] Dear @tsheporamantso, You did a great job adding the background image. But, a single background image is not compatible with different mobile screen sizes, kindly download each piece of the image alone from the Figma design and then try to set each piece to its own position in the different sections of your page.